pkgs/tools/misc/betterdiscord-installer/default.nix

```diff
@@ -1,12 +1,12 @@
 { appimageTools, lib, fetchurl }:
 let
   pname = "betterdiscord-installer";
-  version = "1.0.0-beta";
+  version = "1.1.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://github.com/BetterDiscord/Installer/releases/download/v${version}/Betterdiscord-Linux.AppImage";
-    sha256 = "103acb11qmvjmf6g9lgsfm5jyahfwfdqw0x9w6lmv1hzwbs26dsr";
+    sha256 = "1198fcbd1c966c47c26a0cbfc774131400ef13a67be9faa70d54506610f44b69";
```

Comment:
Incorrect Hash, should be `0sabyh86cl2l1nkzmsbvlq9yy00l2dscggqcdb14fv4n3jyzr60i`